sql/core/src/main/scala/org/apache/spark/sql/jdbc/JdbcDialects.scala

```diff
@@ -117,7 +117,7 @@ abstract class JdbcDialect extends Serializable with Logging {
   def convertJavaTimestampToTimestamp(t: Timestamp): Timestamp = t
 
   /**
-   * Convert java.sql.Timestamp to a LocalDateTime representing the same wall-clock time as the
+   * Convert `java.sql.Timestamp` to a LocalDateTime representing the same wall-clock time as the
```

Comment:
It doesn't matter. Personally, I suggest not changing it.

Reply:
thank your review